server/ai_worker_test.go

```diff
@@ -372,7 +372,10 @@ func createAIJob(taskId int64, pipeline, modelId, inputUrl string) *net.NotifyAI
 		inputFile.InitFromBytes(nil, inputUrl)
 		req = worker.GenSegmentAnything2MultipartRequestBody{ModelId: &modelId, Image: inputFile}
 	case "llm":
-		req = worker.GenLLMFormdataRequestBody{Prompt: "tell me a story", ModelId: &modelId}
+		var msgs []worker.LLMMessage
+		msgs = append(msgs, worker.LLMMessage{Role: "system", Content: "you are a robot"})
```

Comment:
What are these messages? Prompts? Then, shouldn't they come from user's input?

Reply:
These are in tests so that we can run tests with basic inputs.

Reply:
ahh, ok